plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/internal/maven2/metadata/MetadataRebuilder.java

```diff
@@ -59,10 +58,9 @@
 import com.orientechnologies.orient.core.metadata.schema.OType;
 import com.orientechnologies.orient.core.record.impl.ODocument;
 import com.orientechnologies.orient.core.sql.query.OSQLAsynchQuery;
-import com.orientechnologies.orient.core.tx.OTransaction.TXTYPE;
-import org.w3c.dom.Document;
-import org.w3c.dom.NodeList;
-import org.xml.sax.SAXException;
+import org.codehaus.plexus.util.xml.Xpp3Dom;
```

Comment:
@mcculls Thanks. @kellyrob99 Yes, and as @joedragons spotted, there are some strange logging happening to stdout with replaced XML parser. This is just a library, and as we are "tied" to repository metadata modello classes, I think we should be tied to this one too.